docker/genesisTemplate.json

```diff
@@ -38,7 +33,7 @@
     }
   },
   "nonce": "0x0",
-  "timestamp": "0x638F7000",
+  "timestamp": "0x0",
```

Comment:
For my own understanding, is there a reason why this timestamp needs to be 0? It's the `block.timestamp` value for the gensis block, right?

Reply:
yup thats right, I just changed it to be consistent with the warp genesis file we have for `awm_relayer`